aspnetcore-webapi/BuildQueryString/BooksAPI/Controllers/BooksController.cs

```diff
@@ -0,0 +1,18 @@
+using Microsoft.AspNetCore.Mvc;
+
+namespace BooksAPI.Controllers
+{
+    [ApiController]
+    [Route("api/[controller]")]
+    public class BooksController : ControllerBase
+    {
+
+        [HttpGet]
+        public ActionResult Get(string author, string language)
+        {
+            string bookDetails = $"Author: {author}, Language:{language}";
```

Comment:
Use var here
Second option would be to drop the variable altogether and put the string directly in the `return Ok()`

Reply:
I've put the  string directly in the return Ok()